Extension/src/LanguageServer/configurations.ts

```diff
@@ -630,8 +630,7 @@ export class CppProperties {
 
             // Replace all \<escape character> with \\<character>.
             // Otherwise, the JSON.parse result will have the \<escape character> missing.
-            readResults = readResults.replace(/\\/g, '\\\\');
-            readResults = readResults.replace(/\\\\"/g, '\\"'); // Need to revert the change to \".
+            readResults = readResults.replace(/([^\\])\\([^\"\\])/g, '$1\\\\$2'); // Exclude \" and \\.
```

Comment:
This doesn't fix any >2 slashes `\\\` or incorrect end of string slashes `\"`

Reply:
The cases you mentioned were no incorrect, but the `\\` case was broken, leading to no squiggles. I reverted it back to my original fix.